content/en/datadog_cloudcraft/_index.md

```diff
@@ -42,37 +45,73 @@ Cloudcraft's core functionality is its ability to generate detailed architecture
+
+[3]: /security/cloud_security_management
+[10]: /datadog_cloudcraft/overlays#security
+[12]: /security/sensitive_data_scanner
+[13]: /account_management/rbac/permissions/#compliance
+[14]: /getting_started/integrations/azure/
+[15]: https://app.datadoghq.com/integrations?search=azure
+[16]: /integrations/guide/azure-manual-setup/?tab=azurecli#setup
+
+{{% /tab %}}
+{{< /tabs >}}
 
 ## Getting started
 
 To get started using Cloudcraft, use the following steps:
 1. Navigate to [**Infrastructure > Resources > Cloudcraft**][7]. 
 2. A real-time diagram of the resources is displayed in your environment.
 
- **Note**: For environments with more than 10,000 resources, you must filter the diagram by account, region, or tags before it can be displayed.
+**Note**: If your environment has more than 10,000 resources, filter the diagram by account, region, or tags before it can be displayed.
 
-{{< img src="datadog_cloudcraft/getting_started_3.mp4" alt="Video showing getting started in Cloudcraft by selecting the Account, Region, and Resource" video=true;" >}}
+{{< img src="datadog_cloudcraft/getting_started.png" alt="Getting started in Cloudcraft, displaying a list of resources for the selected account and region" style="width:100%;" >}}
 
-**Note**: The account name in the **Account** dropdown originates from your AWS account tags in the AWS integration tile.
+<div class="alert alert-tip">The account name in the <strong>Account</strong> dropdown comes from your AWS account tags in the AWS integration tile. For Azure, the <strong>Azure Subscription</strong> name comes from the subscription name in your Azure integration tile's list of managed subscriptions.
+</div>
 
 ### Group By
 
 With Group By, Cloudcraft divides your diagram into distinct sections based on different group types. This feature offers a clear and organized perspective of your resources, making it especially helpful for visualizing complex cloud environments.
 
-Enable the **Show All Controls** toggle to display the available **Group By** options. You can also remove specific groupings by unchecking options like VPC and Region. To view the current nesting structure and add the Network ACL (Network Access Control List) layer, click the **More** dropdown.
+Enable the **Show All Controls** toggle to display the available **Group By** options. You can remove specific groupings by unchecking options like VPC and Region. To view the current nesting structure and add the Network ACL (Network Access Control List) layer, click the **+ Tags** menu.
 
-{{< img src="datadog_cloudcraft/cloudcraft_group_by_with_ccm.png" alt="Group by feature in Cloudcraft, highlighting the Group By menu" >}}
+{{< img src="datadog_cloudcraft/cloudcraft_group_by_with_network_acl.png" alt="Group by feature in Cloudcraft, highlighting the Group By menu, and the teams tag" >}}
```

Comment:
```suggestion
{{< img src="datadog_cloudcraft/cloudcraft_group_by_with_network_acl.png" alt="Cloudcraft landing page with Group by highlighted, and the teams tag" >}}
```

Reply:
I'm not seeing the teams tag in this screenshot. Does the image need to be updated, or the alt text?